src/main/java/io/xstefank/wildfly/bot/util/PullRequestLogger.java

```diff
@@ -0,0 +1,116 @@
+package io.xstefank.wildfly.bot.util;
+
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class PullRequestLogger {
+    private static final Map<String, Logger> delegates = new HashMap<>();
+    private final String Fqcn;
+    private GHPullRequest pullRequest;
+
+    private PullRequestLogger(String Fqcn) {
+        this.Fqcn = Fqcn;
+    }
+
+    public static PullRequestLogger getLogger(Class<?> loggedClass) {
+        return getLogger(loggedClass.getName());
+    }
+
+    public static PullRequestLogger getLogger(String Fqcn) {
+        delegates.putIfAbsent(Fqcn, Logger.getLogger(Fqcn));
```

Comment:
why do you need to this in getter?

Reply:
I wanted to keep some of the most important methods, see also `log()` or `logf()`.
I can remove unused ones if you want but I wanted to make the most probable to be used in future as well.